core/modules/views_ui/css/views_ui.admin.theme.css

```diff
@@ -27,12 +27,12 @@
 
 /* @group Icons */
 
-.views-admin .icon {
+.views-admin .icon:not(svg) {
```

Comment:
This could be a problem if/when we replace views sprites with SVGs. We can cross that bridge then :D

Reply:
This is (currently) necessary, otherwise the display in the views admin form breaks.
As soon as we replace the custom icon images with our svg files, this will need an update, anyway (lots of CSS code will get obsolete then).